rskj-core/src/test/java/co/rsk/peg/lockingcap/LockingCapSupportImplTest.java

```diff
@@ -0,0 +1,303 @@
+package co.rsk.peg.lockingcap;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.net.utils.TransactionUtils;
+import co.rsk.peg.InMemoryStorage;
+import co.rsk.peg.lockingcap.constants.LockingCapConstants;
+import co.rsk.peg.lockingcap.constants.LockingCapMainNetConstants;
+import co.rsk.peg.storage.StorageAccessor;
+import java.util.Optional;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
+import org.ethereum.core.BlockTxSignatureCache;
+import org.ethereum.core.ReceivedTxSignatureCache;
+import org.ethereum.core.SignatureCache;
+import org.ethereum.core.Transaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class LockingCapSupportImplTest {
+    private final LockingCapConstants constants = LockingCapMainNetConstants.getInstance();
+    private LockingCapSupport lockingCapSupport;
+    private LockingCapStorageProvider lockingCapStorageProvider;
+    private ActivationConfig.ForBlock activations = ActivationConfigsForTest.all().forBlock(0);
+    private SignatureCache signatureCache;
+    private StorageAccessor bridgeStorageAccessor;
+
+    @BeforeEach
+    void setUp() {
+        bridgeStorageAccessor = new InMemoryStorage();
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+        signatureCache = new BlockTxSignatureCache(new ReceivedTxSignatureCache());
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+    }
+
+    @Test
+    void getLockingCap_whenNoPreviousValueExists_ShouldReturnInitialValue() {
```

Comment:
nit:
```suggestion
    void getLockingCap_whenNoPreviousValueExists_shouldReturnInitialValue() {
```
same for the other tests

Reply:
Done! Good catch!